src/constants.zig

```diff
@@ -94,6 +94,7 @@ comptime {
 /// The maximum number of release versions (upgrade candidates) that can be advertised by a replica
 /// in each ping message body.
 pub const vsr_releases_max = config.cluster.vsr_releases_max;
+pub const multiversion_binary_size_max = config.process.multiversion_binary_size_max;
```

Comment:
Is this the binary size of a single release, or of the cumulative multi-version binary? The name implies the latter, but its use in `multiversioning.zig` seems to imply the former. :thinking:

Reply:
It's the latter; but it is used as the former too in release.zig (the size of a single binary <= the size of a pack for buffer limits when reading in).

Not sure where in `multiversioning.zig` it's used in that way? We allocate space for the full multiversion artifact in `init()`.

Reply:
Ah right, nevermind! Though could you specify that it is the cumulative size in `multiversion_binary_size_max`'s doc comment?